### ISSUE TITLE
PAL Color

### ISSUE THREAD

**Issue by aquaticus** [aquaticus/esp32_composite_video_lib#2]
Implement support for PAL color.

**Comment by ChisBread** [comment]
Is there any progress? 
Thank you for your excellent work.

**Comment by aquaticus** [comment]
Nope. I'm stack with NTSC which is simpler (NTSC color signal seems to be OK on scope but no color on the screen).

**Comment by ChisBread** [comment]
https://github.com/siesta0123/DawnOfAV
Hope it helps you a little :D
